clients/guzzle6-adapter.rst

```diff
@@ -1,2 +1,39 @@
-Guzzle6 Adapter
-===============
+Guzzle 6 Adapter
+================
+
+An HTTPlug adapter for the `Guzzle 6 HTTP client`_.
+
+First, create a Guzzle client passing any options you like::
+
+    $config = [
+        // Config params
+    ];
+    $guzzle = new GuzzleHttp\Client($config);
+
+Then create the adapter::
+
+    use Http\Adapter\Guzzle6\Client;
+
+    $adapter = new Client($guzzle);
+
+And use it to send synchronous requests::
+
+    use GuzzleHttp\Psr7\Request;
+
+    $request = new Request('GET', 'http://httpbin.org');
```

Comment:
I think it rather matters in a library, then an application. I would keep both and state that message factory should be used in a library.